### ISSUE TITLE
If both indicators are at the left most point they get stuck

### ISSUE THREAD

**Issue by Haffi112** [boeledi/RangeSlider#5]
If I slide both indicators for the range picker to the left most point I cannot slide the right indicator back to the right.

I'm not sure if this is the intended behaviour but it looks like it could be better. My work around is to detect this and change the value of the right indicator if this happens.

**Comment by raacker** [comment]
It seems that if left and right thumb gets same value because of divisions (the README.md example also has), left thumb overlaps right thumb. Might try to work on this

**Comment by boeledi** [comment]
Hi,  I just released version 1.1.0 which now "normally" corrects this behavior.  Thanks for your contribution

**Comment by Harteg** [comment]
I am still have this problem on 1.1.0:

```
void main(){
  runApp(MaterialApp(
    title: "Test",
    home: Home(),
  ));
}

class Home extends StatefulWidget{
  @override
  _HomeState createState() => _HomeState();
}

class _HomeState extends State<Home> {

  double lower = 0;
  double higher = 1000;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Padding(
        padding: const EdgeInsets.all(16.0),
        child: RangeSlider(
          min: 0,
          max: 1000,
          divisions: 1000,
          lowerValue: lower,
          upperValue: higher,
          valueIndicatorMaxDecimals: 0,
          onChanged: (double newLower, double newHigher) {
            setState(() {
              lower = newLower;
              higher = newHigher;
            });
          },
        ),
      ),
    );
  }
}
```
